snappy/SymServer_RequestHandler.py

```diff
@@ -14,6 +14,7 @@ def prepare(self):
         xForwardIp = self.request.headers.get("X-Forwarded-For")
         self.remoteIP = self.request.remote_ip if not xForwardIp else xForwardIp
         self.requestId = uuid.uuid4()
+        self.debugAllowed = (self.request.remote_ip == "127.0.0.1")
```

Comment:
I wanted to be stricter about what we consider to be the remote IP, since I REALLY do not want outsiders accessing debug commands.